dask_ml/model_selection/_split.py

```diff
@@ -450,19 +452,54 @@ def train_test_split(
 
     if all(isinstance(arr, (dd.Series, dd.DataFrame)) for arr in arrays):
         check_matching_blocks(*arrays)
-        if blockwise is None:
-            blockwise = False
+        if blockwise is False:
+            raise NotImplementedError(
+                "'blockwise=False' is not currently supported for dask DataFrames."
+            )
 
         rng = check_random_state(random_state)
         rng = draw_seed(rng, 0, _I4MAX, dtype="uint")
-        return list(
-            itertools.chain.from_iterable(
-                arr.random_split([train_size, test_size], random_state=rng)
-                for arr in arrays
+        if DASK_GT_2130:
+            if shuffle is None:
+                shuffle = False
+                if not sys.warnoptions:
+                    import warnings
+                    warnings.warn(
+                        message="The shuffle parameter will default to True"
+                                " in the future. Set to False to conserve"
+                                " backwards compatibility.",
+                        category=FutureWarning,
+                    )
+            return list(
+                itertools.chain.from_iterable(
+                    arr.random_split(
+                        [train_size, test_size],
+                        random_state=rng,
+                        shuffle=shuffle,
+                    )
+                    for arr in arrays
+                )
+            )
+        else:
+            if not shuffle:
+                raise NotImplementedError(
+                    f"'shuffle=False' is not supported for DataFrames in"
+                    f" dask versions<2.12.1. Current version is {DASK_VERSION}."
+                )
+            return list(
+                itertools.chain.from_iterable(
+                    arr.random_split([train_size, test_size], random_state=rng)
+                    for arr in arrays
+                )
```

Comment:
Let's just simplify this...

```python
if DASK_213:
    kwargs = dict(shuffle=shuffle)
else:
    kwargs = dict()


return list(
                itertools.chain.from_iterable(
                    arr.random_split([train_size, test_size], random_state=rng, **kwargs)
                    for arr in arrays
                ) 
```

with some care to raise an exception when the user specifies `shuffle=True`, but have an older dask.